### PR TITLE
Run tests in github actions

### PR DESCRIPTION
**Pull request by sfoale**
Migrating from old jenkins

### CODE REVIEW

**Comment by cmccully** [comment]
Do we want to add something to push to pypi on tag?

**Comment by sfoale** [comment]
> Do we want to add something to push to pypi on tag?

I did wonder that. Do we do this with any other package? I looked at the secrets and couldn't find anything for pypi so figured maybe not?

**Comment by mgdaily** [comment]
> > Do we want to add something to push to pypi on tag?
> 
> I did wonder that. Do we do this with any other package? I looked at the secrets and couldn't find anything for pypi so figured maybe not?

The OCS does. Here's an example for the ingester library. We'll just need to figure out the secrets as you say
https://github.com/observatorycontrolsystem/ingester/blob/7f1e5c827f908328a149194a623301aaebe5737c/.github/workflows/build.yaml#L40

**Comment by sfoale** [comment]
> > > Do we want to add something to push to pypi on tag?
> > 
> > 
> > I did wonder that. Do we do this with any other package? I looked at the secrets and couldn't find anything for pypi so figured maybe not?
> 
> The OCS does. Here's an example for the ingester library. We'll just need to figure out the secrets as you say https://github.com/observatorycontrolsystem/ingester/blob/7f1e5c827f908328a149194a623301aaebe5737c/.github/workflows/build.yaml#L40

Thanks - that helped. I've added a secret for the pypi api token and a push on tag action.

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build package and publish to PyPI
+      env:
+        TWINE_USERNAME: __token__
+        TWINE_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
```
Comment (mgdaily) [inline .github/workflows/build.yaml:57]: Looks good. I'm assuming you got the API token for the LCOGT PyPI account?

**Review by mgdaily — APPROVED** [review]

File: .github/workflows/build.yaml (hunk 1 above)
Comment (sfoale) [reply .github/workflows/build.yaml:57]: That's right.

**Pull request merged**